shared/defs/gameObjects/unlockDefs.js

```diff
@@ -1,8 +1,45 @@
+import { CrosshairDefs } from "./crosshairDefs";
+import { EmotesDefs } from "./emoteDefs";
+import { HealEffectDefs } from "./healEffectDefs";
+import { OutfitDefs } from "./outfitDefs";
+
+const allowedHealEffects = Object.keys(HealEffectDefs);
+const allowedMeleeSkins = ["fists", "knuckles_rusted", "knuckles_heroic", "karambit_rugged", "karambit_prismatic", "karambit_drowned", "bayonet_rugged", "bayonet_woodland", "huntsman_rugged", "huntsman_burnished", "bowie_vintage", "bowie_frontier"];
+
+/**
+ * Checks if an item is present in the player's loadout
+ * @param {string} item
+ * @param {"boost" | "heal" | "melee" | "emote" | "outfit"} category
+ * @returns { boolean }
+ */
+export function isItemInLoadout(item, category) {
+    switch (category) {
+    case "outfit": {
+        return item in OutfitDefs;
```

Comment:
uh should probably check if the outfit is in the unlock defs otherwise players can send Ghillie suit as their skin and it will accept it